freedomofpress/securedrop
# Lints all Python files

## Status

Ready for review.

## Description of Changes

Fixes #2933.

Lints the entire repo with `flake8`, omitting only:

  * `config.py` (intending specifically `securedrop/config.py`)
  * `.venv/` (the virtualenv for prod environments, containing pip-installed libs we don't control)

Trimmed out a few unused files, e.g. `install_files/ansible-base/roles/restore/files/0.3_restore.py`, but only if not used anywhere. Additional files can be removed, but would require QA, so opened #2943 to track one example.

## Testing

1. Run `make flake8` and confirm it passes.
2. Choose Python files at various locations throughout the repository, introduce formatting violations.
3. Repeat.

See if you can find any Python files that are not caught by this strategy. 

## Deployment

Minor: as long as the changes presented are genuinely lint-resolving, and not functional changes, then no concern for prod deployments. CI should catch any egregious mistakes.

## Checklist

### If you made changes to the app code:


### If you made changes to the system configuration:


### If you made changes to documentation:
